master/buildbot/status/worker.py

```diff
@@ -41,6 +42,24 @@ def __init__(self, name):
         self.graceful_callbacks = []
         self.pause_callbacks = []
         self.connect_times = []
+        self.info = Properties()
+
+    def setInfo(self, info):
```

Comment:
Don't add new API to status classes. They are meant to disappear.
Please move this to worker class.